src/Components/EnvironmentPicker/EnvironmentPickerManager.ts

```diff
@@ -141,6 +146,7 @@ export const getContainerNameFromUrl = (containerUrl: string) => {
 };
 
```

Comment:
![image](https://user-images.githubusercontent.com/57726991/186236917-6e9e8b72-2104-4655-ac7e-156661ca1429.png)

Reply:
had commented and marked as "resolved" this on the original PR